library/lua/gui.lua

```diff
@@ -913,10 +913,10 @@ local BASE_FRAME = {
 }
 
 local function make_frame(name, double_line)
-    local texpos = dfhack.textures['get'..name..'BordersTexposStart']()
     local tp = function(offset)
+        local texpos = dfhack.textures.getAsset('hack/data/art/border-'..name:lower()..'.png', offset)
```

Comment:
I suggest making the strings that this function is called with lower case instead of paying the cost of lower casing each time this function is called

Reply:
Yeah, we can avoid addition cost here.
Can it be in the next iteraction of refactor? (If it will be assets init in lua)